lib/voyager_helpers/liberator.rb

```diff
@@ -396,32 +388,40 @@ def get_records_from_barcode(barcode, recap=false)
       # @param date [String] in format yyyy-mm-dd hh24:mi:ss.ffffff timezone_hourtimezone_minute (e.g., 2017-04-05 13:50:25.213245 -0400)
       # @return [Array]
       def updated_recap_barcodes(date)
+        items = []
         barcodes = []
-        query = VoyagerHelpers::Queries.recap_update_bib_barcodes
         connection do |c|
+          query = VoyagerHelpers::Queries.recap_update_bib_items
           cursor = c.parse(query)
           cursor.bind_param(':last_diff_date', date)
           cursor.exec()
           while row = cursor.fetch
-            barcodes << row.first
+            items << row.first
           end
           cursor.close()
-          query = VoyagerHelpers::Queries.recap_update_holding_barcodes
+          query = VoyagerHelpers::Queries.recap_update_holding_items
           cursor = c.parse(query)
           cursor.bind_param(':last_diff_date', date)
           cursor.exec()
           while row = cursor.fetch
-            barcodes << row.first
+            items << row.first
           end
           cursor.close()
-          query = VoyagerHelpers::Queries.recap_update_item_barcodes
+          query = VoyagerHelpers::Queries.recap_update_item_items
           cursor = c.parse(query)
           cursor.bind_param(':last_diff_date', date)
           cursor.exec()
           while row = cursor.fetch
-            barcodes << row.first
+            items << row.first
           end
           cursor.close()
+          items.each do |item|
+            item_statuses = get_item_statuses(item, c)
+            if (['In Process'] & item_statuses.keys).empty?
```

Comment:
I did that in case other statuses needed to be added at a later point, but I'm pretty certain at this point that they won't be. Will do.